gui/wxpython/datacatalog/tree.py

```diff
@@ -761,7 +798,7 @@ def OnPasteMap(self, event):
                                                    element=self.copy_layer[i].data['type'])
                     if not new_name:
                         continue
-                gisdbase = gisenv()['GISDBASE']
+                gisdbase = self.selected_grassdb[0]
```

Comment:
Here you need to distinguish between the database from where we reproject and to which database we reproject.